tests/phpunit/tests/blocks/serialize.php

```diff
@@ -54,4 +54,23 @@ public function test_serialized_block_name() {
 		$this->assertSame( 'example', strip_core_block_namespace( 'core/example' ) );
 		$this->assertSame( 'plugin/example', strip_core_block_namespace( 'plugin/example' ) );
 	}
+
+	public function test_callback_argument() {
```

Comment:
Similar notes as @costdev added in https://github.com/WordPress/wordpress-develop/pull/5186#pullrequestreview-1620391188. I believe it can use `#59313` as the ticket number.

Reply:
Good point!

I actually filed [`#59327`](https://core.trac.wordpress.org/ticket/59327) just for this PR 😄

Reply:
b4dc0e5d7f